careerfairsystems/arkad-app-2018
# Rename filter reducer to favorite

## Change description

Rename filter reducer to favorite and move toggle favorites to company reducer so that it's not persisted.

## How to verify

Run the project and add a favorite. Reload the app and confirm that the state is persisted. Also press show favorites and reload the app. Confirm that the state is NOT persisted.

## Issues fixed

This fixes #44.
